plugins/examples/details-view/src/index.tsx

```diff
@@ -26,3 +28,36 @@ registerDetailsViewSection(({ resource }: DetailsViewSectionProps) => {
   }
   return null;
 });
+
+registerDetailsViewHeaderActionsProcessor({
+  id: 'my-custom-header-actions-processor',
+  processor: (resource, actions) => {
```

Comment:
Some commentary on what this does and why would be useful. Passing in a named function(still inline), instead of an anonymous one could help with this.

Reply:
I fixed this by updating the ID so it describes better what it does.